biogrid/apps/biogrid-mvp/src/app/pages/info-page/info-page.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './info-page.css';
+import Visual from './EnergyFlowVisual.png';
+
+export const InfoPage = () => {
+
+
+  return (
+    <div className="info-page">
+      <h1>The Logic</h1>
+
+      <img src={Visual} id="EnergyFlow"></img>
+
+      <p>Firstly, the grid items <b>(small batteries(s_b), large batteries(l_b) solar panels(s), and buildings(B))</b> will be randomly placed in the town. 
+      The grid items include the <b>grid(g)</b> itself as well.
+      </p>
+
+      <p>Then the grid items will be connected by a <b>directed graph G</b>, which connects them from the <b>grid</b> to each item except the solar panels i.e <b>G(g, s_b), G(g, l_b), G(g, B)</b>. 
```

Comment:
I think this sentence is a bit confusing. Maybe
`Each grid item except the solar panels is connected to grid g by a directed graph (G)`